01.md

```diff
@@ -6,11 +6,20 @@ Nostr Protocol
 
 `draft` `mandatory`
 
-This NIP defines the mandatory part of the Nostr protocol. New NIPs may add new optional (or mandatory) fields, messages, and features to the structures and flows described here.
+This NIP defines the mandatory part of the Nostr protocol. New NIPs may add new fields, message kinds, and features to the structures and flows described here.
 
-# Events
+# Signed Events
 
-Event is the only object type available. It is a hashed and signed payload in the following format:
+Events are the only data type transferred across the network. They consist in 4 main attributes: 
```

Comment:
5 main attributes

Reply:
3 m'lord, 3

Reply:
6 if you count `id`?

Reply:
5 things are hashed:  pubkey, created_at, kind, tags, and content.   The other 2 are generated:  id, sig